Source/ServerTrustEvaluation.swift

```diff
@@ -358,8 +412,53 @@ public extension SecTrust {
     var isValid: Bool {
```

Comment:
Pretty sure this is only used in the test suite. Any harm in moving this API into the test suite directly to avoid confusion? I was thinking this was being called since there's a duplicate `isValid` function.

Reply:
I've moved it, though it used some convenience API I had to redo manually.